Build/Build.cs

```diff
@@ -0,0 +1,140 @@
+using System;
+using System.Linq;
+using Nuke.Common;
+using Nuke.Common.Execution;
+using Nuke.Common.IO;
+using Nuke.Common.ProjectModel;
+using Nuke.Common.Tooling;
+using Nuke.Common.Tools.DotNet;
+using Nuke.Common.Tools.GitVersion;
+using Nuke.Common.Tools.NUnit;
+using Nuke.Common.Tools.Xunit;
+using Nuke.Common.Utilities.Collections;
+using static Nuke.Common.EnvironmentInfo;
+using static Nuke.Common.IO.FileSystemTasks;
+using static Nuke.Common.IO.PathConstruction;
+using static Nuke.Common.Tools.DotNet.DotNetTasks;
+using static Nuke.Common.Tools.NUnit.NUnitTasks;
+using static Nuke.Common.Tools.Xunit.XunitTasks;
+
+[CheckBuildProjectConfigurations]
+[UnsetVisualStudioEnvironmentVariables]
+[DotNetVerbosityMapping]
+class Build : NukeBuild
+{
+    /// Support plugins are available for:
+    ///   - JetBrains ReSharper        https://nuke.build/resharper
+    ///   - JetBrains Rider            https://nuke.build/rider
+    ///   - Microsoft VisualStudio     https://nuke.build/visualstudio
+    ///   - Microsoft VSCode           https://nuke.build/vscode
+    public static int Main() => Execute<Build>(
```

Comment:
Does not work locally for me, as it compiles in debug mode, but tries to pack in release mode.
@dennisdoomen you had some opinions about default invocation of `build.ps1`?

Reply:
Everything should be compiled in _release_ mode, except for the unit tests. We need the full symbols to be able to get proper identifier names.